lms/templates/instructor/instructor_dashboard_2/course_info.html

```diff
@@ -70,12 +70,19 @@
       <b>${_("No")}</b>
       %endif
     </li>
-
     <li class="field text is-not-editable" id="field-grade-cutoffs">
       <label for="start-date">${_("Grade Cutoffs:")}</label>
       <b>${ section_data['grade_cutoffs'] }</b>
     </li>
   </ul>
+
+  %if settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD', ''):
+      <p>
+        ## Translators: git is a version-control system; see http://git-scm.com/about
+        ${_("View detailed Git import logs for this course:")}
+        <a href="${ section_data['detailed_gitlogs_url'] }">${_("Git import logs")}</a>
```

Comment:
I think this would be better as one sentence, both from internationalization and user perspective.

_("View detailed Git import logs for this course {link_start}by clicking here{link_end}")

or maybe,

_("To see the status of past course imports, {link_start}view detailed Git import logs for this course{link_end}")

then `_("text").format(link_start='<a href="${ section_data['detailed_gitlogs_url'] }">', link_end='</a>')`